cci_nepal/analysis/wp2_data_scoping/database_sample/nepal_flood_landslide_descriptive.py

```diff
@@ -0,0 +1,1006 @@
+len(
+    data_df[
+        "Are there any suggestion you want to give to Nepal Red Cross society to improve the relief distribution program in the future Mention if any"
+    ].unique()
+)
+
+# %% [markdown]
+# # Some thoughts arising from the data
+# 1. Which communication means adopted by nepal RC proved more effective for the affected communities?
+# 2. Of the financial aid provided, how can one prioritize the needs of the communities based on how the aid was used?
+#
+# 3. based on the time taken to arrive at the relief distribution centers, how convenient is it to reduce the number of people who travelled between 3-4 hours to get the aid? Can new centers that would reduce this travel time be created?
+#
+# 4. For the communication modes used, were resources equitably distributed to the different channels?
+#
+
+# %%
+# replace the 9 null values with the mode for the specific column on mode of communication.
+data_df["How were you notified about the relief delivery date"] = data_df[
+    "How were you notified about the relief delivery date"
+].fillna(
+    "        From community representatives TeachersCommunity leaders and peoples representatives"
+)
+communication_modes = data_df[
+    "How were you notified about the relief delivery date"
+].unique()
+
+# %%
+# check for duplicate records in the data - none found
+data_df.duplicated().sum()
+
+# %%
+# skewness of data
+data_df.skew(axis=0, skipna=True)
+
+# %%
+right_skew = (42 / 50) * 100
+right_skew
+
+# %%
+left_skew = (4 / 50) * 100
+left_skew
+
+# %%
+data_df["Are you satisfied with the relief distribution process"].isnull().sum()
+
+# %%
+# feedback on how people felt about the distribution process.
+data_df["Are you satisfied with the relief distribution process"].value_counts()
+
+# %% [markdown]
+# ## What accounts for the fact that some districts are more represented than others?
+
+# %%
+district_counts = data_df["District name"].value_counts()
+
+# %%
+district_counts
+
+# %%
+population.District.unique()
+
+# %%
+# extract population of distrricts in the the sample data
+district_population = {}
+for name in data_df["District name"].unique():
+    if name in population.District.unique():
+        district_population[name] = population.Value[population.District == name]
+        print(population.Value[population.District == name])
+
+# %%
+district_counts[:14]
+
+# %%
+# district names and their frequencies in the sample data
+names = [
+    "Lamjung",
+    "Achham",
+    "Kailali",
+    "Sindhupalchok",
+    "Myagdi",
+    "Dhading",
+    "Dolakha",
+    "Sankhuwasabha",
+    "Gulmi",
+    "Jajarkot",
+    "Bajura",
+    "Baglung",
+    "Darchula",
+    "Arghakhanchi",
+    "8 others",
+]
+counts = [50, 37, 32, 28, 28, 26, 25, 18, 16, 15, 11, 9, 7, 7, 14]
+
+
+# %%
+# plotting proportions of informants from the different districts.
+fig2, ax2 = plt.subplots()
+ax2.pie(counts, labels=names, autopct="%1.1f%%", shadow=True, startangle=90)
```

Comment:
I would say this is too many values for a pie chart to be clearly interpretable.

Reply:
I maintained the pie chart since I left it on the report. I could completely remove it from both documents given the map showing the districts was introduced in the report. I think it's what I should do.